.licenserc.yaml

```diff
@@ -21,17 +21,62 @@ header:
   license:
     spdx-id: Apache-2.0
     copyright-owner: Apache Software Foundation
+    content: | # `license` will be used as the content when `fix` command needs to insert a license header.
```

Comment:
Could we use owner to replace all these?

Check the more repo, https://github.com/apache/skywalking/blob/master/.licenserc.yaml#L22

@kezhenxu94